packages/logic-utils/api.d.ts

```diff
@@ -172,3 +172,11 @@ export function convertJSDocToMarkdown(
  * @param link
  */
 export function getLink(model: UI5SemanticModel, link: string): string;
+
+/**
+ * Split possibly qualified tag or attribute name to namespace and name
+ * @param qName
+ */
+export function splitQNameByNamespace(
```

Comment:
- This possibly belongs in XML-Tools
- `QName` is ambiguous, is this "x.y.z" (regular fqn) or XMLNs fqn "foo:bar"
  There is no way to tell from the docs.
- I know its the xmlns name... so in that case it should use xmlns terms, e.g:
  - ns --> **prefix**

Reply:
> * ~This possibly belongs in XML-Tools~

Actually I agree, but I don't think I'll have time to move it this sprint.

> * `QName` is ambiguous, is this "x.y.z" (regular fqn) or XMLNs fqn "foo:bar"
>   There is no way to tell from the docs.

I'll improve the comment to make it clearer that it's related to xml qualified names.

> * I know its the xmlns name... so in that case it should use xmlns terms, e.g:
>   
>   * ns --> **prefix**

Ok

Reply:
> * I know its the xmlns name... so in that case it should use xmlns terms, e.g:
>   
>   * ns --> **prefix**

On the xml element it's called `ns`, even though it's the prefix and not the resolved namespace.

Reply:
Changed

Reply:
> On the xml element it's called ns, even though it's the prefix and not the resolved namespace.

Yes that is a mistake I've made, but fixing that would be a serious breaking change